recipes/ipypdf/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: ipypdf
+  version: {{ version }}
+
+source:
+  url: https://github.com/joelstansbury/ipypdf/archive/0206fb7.tar.gz
+  sha256: ec1e9cfd2db9e45963e2664d86acfdaf7b5c4568e654ceb3c6f60d7a29f65f3d
+
+build:
+  script: {{ PYTHON }} -m pip install -vv . --no-deps
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - pip
```

Comment:
does it actually use `pip` at runtime? this can create unwanted side-effects

Reply:
It uses pip to install layoutparser when the user clicks a button. The alternative is that `pip install layoutparser...` goes into the installation instructions, which I'm open to, but I like the `click to install` option.

Reply:
I mean, your code, your rules, but your users are not going to thank you when some random 75mb package and its very complex dependencies fail to install on windows inside a click handler.  Again, the win is:
- handling it being missing, and do it another day
- getting it into conda-forge so `conda install ipypdf` and you can get on an airplane and everything works.